experiments/metrics/config_overrides/quad_track_2d/variant.yaml

```diff
@@ -1,15 +1,21 @@
 # vary quad mass, nominal M 0.027
+xlabel: M
 reference: 0.027
 variants: [0.021, 0.024, 0.027, 0.03, 0.033] 
```

Comment:
What is this variant stuff? I'm guessing its new? Will keep looking through the PR.